src/main/java/com/wavefront/opentracing/WavefrontSpan.java

```diff
@@ -211,7 +211,7 @@ private synchronized WavefrontSpan updateSpanLogsInternal(
     }
     if (fields != null) {
       Map<String, String> finalFields = fields.entrySet().stream().collect(
-          toMap(Map.Entry::getKey, entry -> entry.getValue().toString()));
+          toMap(Map.Entry::getKey, entry -> entry.getValue() != null ? entry.getValue().toString() : ""));
```

Comment:
Nit: do you mind replacing the ternary with `Objects.toString(entry.getValue(), "")`?

Reply:
@oppegard 
Yes, makes sense, updated. 
Thanks!